package.json

```diff
@@ -17,7 +17,7 @@
     "test:playWright": "npx playwright test",
     "test:playWrightWithUpdate": "npx playwright test --update-snapshots",
     "coverage": "vitest run --coverage",
-    "precommit": "concurrently \"npx lint-staged\" \"npm:build\"",
+    "precommit": "npx lint-staged && npm run build",
```

Comment:
I propose to left only lint-staged in precommit, and run npm run build in the end of prepare-commit-msg. Cuz it's quite sad to wait for build end just to know that you failed commit naming.

Reply:
Implemented